5/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN set -eux; \
     addgroup -S -g 1000 scanner-cli; \
     adduser -S -D -u 1000 -G scanner-cli scanner-cli; \
     apk add --no-cache --virtual build-dependencies wget unzip gnupg; \
-    apk add --no-cache git python3 py3-pip py3-pylint bash shellcheck "nodejs>=18" openjdk17-jre curl musl-locales musl-locales-lang; \
+    apk add --no-cache git bash shellcheck "nodejs>=18" openjdk17-jre musl-locales musl-locales-lang; \
```

Comment:
Did y'all mean to remove curl from the image? Wondering if we need to update our pipelines to install `curl` now, or this was unintentional